bus.js

```diff
@@ -0,0 +1,205 @@
+  for (var i = 0; i < 3; i++) {
+    var nextPhoto = photos.splice(getRandomIndex(photos), 1);
+    photosOnScreen = photosOnScreen.concat(nextPhoto);
+    el = document.getElementById('' + i);
+
+    el.src = nextPhoto[0].filePath;
+    nextPhoto[0].displayed++;
+  }
+}
+function photoSelector(event) {
+  photosOnScreen[event.target.id].clicks++;
+  displayPhotos();
+  clicks++;
+
+  if (clicks === 25) {
+    var imagEl = document.getElementById('images');
+    imagEl.style.display = 'none';
+    displayChart();
+    displayChartTwo();
+    getTable();
+    try {
+      localStorage.photos = JSON.stringify(photos);
+    } catch (error) {
+      console.log('something went wrong', error);
+    }
+  }
+}
+
+
+function displayChart() {
+  photos = photos.concat(photosOnSecondToLastScreen);
+  photos = photos.concat(photosOnScreen);
+  photos = photos.concat(photosOnPreviousScreen);
+  var chartLabel = [];
+  var clicks = [];
+  var displays = [];
+  for (var i = 0; i <photos.length; i++) {
+    chartLabel.push(photos[i].name);
+    clicks.push(photos[i].clicks);
+    displays.push(photos[i].displayed);
+  }
+
+  var canvas = document.getElementById('chart-canvas1');
+  var ctx = canvas.getContext('2d');
+  var data = {
+    labels: chartLabel,
+    datasets:[{
+      label: 'Clicks',
+      backgroundColor:'#2F343B',
+      borderWidth: 1,
+      data: clicks},
+    {
+      label: 'Displayed',
+      backgroundColor: '#703030',
+      borderWidth: 1,
+      data: displays,}
+    ]
+  };
+  canvas.height = '500';
+  canvas.width = '500';
+  var myBarChart = new Chart(ctx, {
+    type: 'bar',
+    data: data,
+  });
+}
+function displayChartTwo() {
+  var chartLabel = [];
+  var percent= [];
+  for (var i = 0; i <photos.length; i++) {
+    chartLabel.push(photos[i].name);
+    percent.push(Math.floor((photos[i].clicks/photos[i].displayed)*100));
+  }
+  var canvas = document.getElementById('chart-canvas2');
+  var ctx = canvas.getContext('2d');
+  var data = {
+    labels: chartLabel,
+    datasets:[{
+      label: '% of Clicks when Viewed',
+      backgroundColor:'#E3CDA4',
+      borderWidth: 1,
+      data: percent,}
+    ]
+  };
+  canvas.height = '500';
+  canvas.width = '500';
+  var myBarChart = new Chart(ctx, {
+    type: 'bar',
+    data: data,
+  });
+}
+
+
+function getTable(){
+
+  var result = document.getElementById('table-data');
+  var table = document.createElement('table');
+  table.id = table;
+  var titleRow = document.createElement('tr');
+  var titleItem = document.createElement('th');
+  titleItem.textContent = 'Item';
+  var titleViews = document.createElement('th');
+  titleViews.textContent = 'Views';
+  var titleClicks = document.createElement('th');
+  titleClicks.textContent = 'Clicks';
+  var titlePercetage = document.createElement('th');
+  titlePercetage.textContent = '% of Clicks When Viewed';
+  var titleRecomended = document.createElement('th');
+  titleRecomended.textContent = 'Recomended';
```

Comment:
You might re-organize these to either group var declarations together and property assignments together or add some blank lines between each to visually separate things.